web/pgadmin/dashboard/static/js/SystemStats/Storage.jsx

```diff
@@ -289,17 +254,17 @@ export default function Storage({preferences, sid, did, pageVisible, enablePoll=
             const di_info_obj = data['di_stats'];
             for (const key in di_info_obj) {
               di_info_list.push({ 
-                icon: '', 
-                file_system: di_info_obj[key]['file_system']?di_info_obj[key]['file_system']:'null',
-                file_system_type: di_info_obj[key]['file_system_type']?di_info_obj[key]['file_system_type']:'null',
-                mount_point: di_info_obj[key]['mount_point']?di_info_obj[key]['mount_point']:'null',
-                drive_letter: di_info_obj[key]['drive_letter']?di_info_obj[key]['drive_letter']:'null',
-                total_space: di_info_obj[key]['total_space']?formatBytes(di_info_obj[key]['total_space']):'null',
-                used_space: di_info_obj[key]['used_space']?formatBytes(di_info_obj[key]['used_space']):'null',
-                free_space: di_info_obj[key]['free_space']?formatBytes(di_info_obj[key]['free_space']):'null',
-                total_inodes: di_info_obj[key]['total_inodes']?di_info_obj[key]['total_inodes']:'null',
-                used_inodes: di_info_obj[key]['used_inodes']?di_info_obj[key]['used_inodes']:'null',
-                free_inodes: di_info_obj[key]['free_inodes']?di_info_obj[key]['free_inodes']:'null',
+                icon: gettext(''), 
+                file_system: di_info_obj[key]['file_system']?gettext(di_info_obj[key]['file_system']):gettext('null'),
```

Comment:
gettext will not work on variables. it works on plain strings.

Reply:
Little confused here, both empty string `''` and `null` are string only.

Reply:
null is not string - null is null. And icon is used internally. not every string should be wrapped. only visible on the UI.

Reply:
okay, I got it for icon, but as you can see currently I'm setting value as null string `'null'`, so should I change it to null object?
used_inodes: di_info_obj[key]['used_inodes']?di_info_obj[key]['used_inodes']:null,

or just keep it like this:
used_inodes: di_info_obj[key]['used_inodes']?di_info_obj[key]['used_inodes']:'null',

Reply:
`null` doesn't mean anything. I would rather keep it blank. You can use empty string instead of null.